### DISCUSSION TITLE
copilot

### DISCUSSION THREAD

**Discussion by ghost** [community/community#123695] [Copilot Conversations]
### Select Topic Area

Question

### Body

how to review history github copilot?

**Comment by nnntan** [comment]
can't review history copilot

**Comment by ebndev** [comment]
Hi @coffeedev4am & @OrcinusDev,

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
